doc/langdef.md

```diff
@@ -827,6 +827,12 @@ values. The ordering operators obey the usual algebraic properties, i.e. `e1 <=
 e2` gives the same result as `!(e1 > e2)` as well as `(e1 < e2) || (e1 == e2)`
 when the expressions involved do not have side effects.
 
+### Overflow
```

Comment:
Could we add a test to ensure negating the minimum `int` value will overflow?

Reply:
Done. Thanks, good catch. In cel-go it wraps back to itself, but for consistency we should expect an error.